backend/submodule/dsp.py

```diff
@@ -19,32 +19,14 @@ class DSP_Mode(Enum):
 
 @dataclass
 class DSP_output:
-    dsp_blocks_used : int = field(default=0)
-    clock_frequency : int = field(default=100000000)
+    dsp_blocks_used : float = field(default=0.0)
```

Comment:
What's the reason to use float instead of integer?

Reply:
This calculated value will become a fraction number based one certain user input conditions. See below:-
![image](https://github.com/os-fpga/rapid_power_estimator/assets/12110149/d9e67982-d1cf-4366-80bc-3869f513d51d)